py/aou_workbench_client/cdr/model.py

```diff
@@ -7,6 +7,7 @@
 
 from .wrapper import RelatedTableWrapper
 import pandas as pd
+from IPython.display import display, HTML
```

Comment:
IPython should be added to requirements.txt

Reply:
Yup, did that. Thanks for the reminder.

(In a follow-up PR I'm going to set up circle integration.)